mmv1/third_party/terraform/services/iap/resource_iap_settings_test.go

```diff
@@ -0,0 +1,106 @@
+package iap_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccIapSettings_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckIapSettingsDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccIapSettings_basic(context),
+			},
+			{
+				ResourceName:      "google_iap_settings.iap_settings",
+				ImportState:       true,
+				ImportStateVerify: true,
```

Comment:
`ImportStateVerifyIgnore: []string{"access_settings.0.workforce_identity_settings.0.oauth2.0.client_secret"}` should be added after this line to fix the test.